lib/index.js

```diff
@@ -43,6 +43,12 @@ const accentsRule = new FunctionRule((s1, s2) => {
   return [normalize(s1prime, 'NFC'), normalize(s2prime, 'NFC')];
 });
 
+const wordPrefixRuleCreator = word =>
+  new FunctionRule((s1, s2) => {
+    const regexp = new RegExp(`^${word}\\s+`, 'i');
```

Comment:
I think you could leave out the case-insensitivity flag and if case-insensivity is desired, it can be achieved by adding the `CAPITALIZATION` rule prior to word prefix rules. That would make word prefix rules a little more composable, as they can either be case-sensitive or not. NBD, though.

Reply:
Good point. We do ignore capitalization within our rules, so SGTM to leave it out here!